### DISCUSSION TITLE
Can TB-RK1808S0 be used as a substitute for Google Coral TPU?

### DISCUSSION THREAD

**Discussion by wxuan1990** [blakeblackshear/frigate#9286] [Ask A Question] [answered]
Currently, with tariffs and such, the Google Coral TPU is estimated to be about 100 dollars, while the TB-RK1808S0 is about 44 dollars. They seem similar in terms of functionality. Can I use the TB-RK1808S0? I plan to use it on an x86 platform.

The performance parameters of TB-RK1808S0 are as follows:

SoC: Rockchip RK1808 dual-core Cortex-A35 processor with NPU
AI Inference Performance: 3 TOPS for INT8, 300 GOPS for INT16, 100 GOPS for FP16
System Memory: 1GB LPDDR
Storage: 8GB eMMC flash
Host Interface: USB 3.0 type-A port
Power Supply: Through USB port
Dimensions: 82 x 31 x 13 mm

![RK1808S0_size](https://github.com/blakeblackshear/frigate/assets/121547018/96959675-e1a1-407f-a084-d4196c87105d)

**Comment by NickM-27** [comment, marked as answer]
it is currently not supported, I spent a good bit of time with one but could not get something that worked very well, perhaps support will be added by a community member
